scripts/haddocs.sh

```diff
@@ -49,6 +50,10 @@ elif [ ${REGENERATE} != "false" ]; then
   cabal haddock "${HADDOCK_OPTS[@]}" ${REGENERATE}
 fi
 
+if [[ !( -d ${OUTPUT_DIR} ) ]]; then
+  mkdir ${OUTPUT_DIR}
+fi
+
```

Comment:
Couldn't this be replaced with `mkdir -p ${OUTPUT_DIR}`?

Reply:
Yes, that's better.

Reply:
The point of `-p` is that you no longer need to check whether it exists or not.